src/Component/colorPalette/colorPalette.css

```diff
@@ -23,7 +23,7 @@
 }
 @media only screen and (min-device-width: 320px) and (max-device-width: 880px) and (-webkit-min-device-pixel-ratio: 2) {
   .colors-selector-container {
-    top: 39rem;
+    top: 46rem;
     left: 17.6rem;
     z-index: 120;
```

Comment:
is there any specific reason for giving it such a high z-index, if not you can try using lower values!! and that should work the same!!

Reply:
okay will do this